src/NUnitEngine/nunit.engine/Runners/MasterTestRunner.cs

```diff
@@ -429,9 +435,20 @@ private TestEngineResult RunTests(ITestEventListener? listener, TestFilter filte
 
             _eventDispatcher.OnTestEvent(startRunNode.OuterXml);
 
+            if (_testRunTimeout > 0)
+            {
+                _testRunTimer = new Timer(_testRunTimeout);
+                _testRunTimer.Elapsed += OnTestRunTimeout;
+                _testRunTimer.AutoReset = false;
+                _testRunTimer.Enabled = true;
```

Comment:
I prefer using object initializers where possible.  

```suggestion
                _testRunTimer = new Timer(_testRunTimeout)
                {
                    AutoReset = false,
                    Enabled = true,
                };
                _testRunTimer.Elapsed += OnTestRunTimeout;
```

Reply:
I like it too... I was experimenting with ordering. Will update.